.github/workflows/default.yml

```diff
@@ -74,11 +131,18 @@ jobs:
         run: goveralls -coverprofile=covprofile -service=github
       - name: Delete container
         id: delete
-        if: ${{ always() && steps.create.outputs.container_id }}
-        uses: Scalr/gh-action-revizor@master
-        with:
-          command: delete
-          container_id: ${{ steps.create.outputs.container_id }}
+        if: ${{ always() }}
+        shell: bash
+        run: |
+          echo "::group::Delete test app"
+          docker run --rm \
+          -w /fatmouse \
+          -v $PWD/fatmouse:/fatmouse \
+          -v ~/.scalr-labs:/etc/scalr-labs \
+          fatmouse/python-builder:master \
+          python -u clickfile.py te rm \
+          --no-wait ${{ env.RUN_TAG }}
```

Comment:
@maratkomarov should we delete broken tev2 containers for provider tests immediately or introduce some delay like we currently have with e2e tests (containers are scheduled to be removed in 30 mins)?

Reply:
@abalakh no need for this. we have everything logged.

Reply:
@maratkomarov yes, but why not, it will make debug easier. I think delay in 30m is good options.